spec/services/payments/lose_dispute_service_spec.rb

```diff
@@ -0,0 +1,102 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe Payments::LoseDisputeService, type: :service do
+  subject(:lose_dispute_service) { described_class.new(payment:) }
+
+  describe "#call" do
+    context "when payment does not exist" do
+      let(:payment) { nil }
+
+      it "returns a failure" do
+        result = lose_dispute_service.call
+
+        aggregate_failures do
```

Comment:
We already aggregate_failures by default, not needed to add it explicitly.

Reply:
forgot to rmeove!